api/http/management_filetransfer.go

```diff
@@ -407,19 +407,22 @@ func (h ManagementController) downloadFileResponseProcessMessage(c *gin.Context,
 func (h ManagementController) DownloadFile(c *gin.Context) {
 	params, statusCode, err := h.getFileTransferParams(c)
 	if err != nil {
+		log.FromContext(c).Error(err)
```

Comment:
FYI: `gin.Context != gin.Context.Request.Context`. IOW you won't get the values attached to the request context (request id, subject id etc.).
And can you please create a logger variable at the start of the function?

Reply:
Thanks @alfrunes, I force-pushed, can you check again?